docker/cellranger/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:16.04
+
+## Install Variables
+ENV pyversion 2.7.13
+
+## Python
+RUN apt-get update && apt-get -y install build-essential checkinstall libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libc6-dev libbz2-dev wget make && wget https://www.python.org/ftp/python/${pyversion}/Python-${pyversion}.tgz && tar -xvf Python-${pyversion}.tgz
```

Comment:
Does it really fail with the 2.7.12 that comes with ubuntu 16.04?

Reply:
Hmm I haven't tried! Cellranger lists Python 2.7.13 under their build dependencies: https://github.com/10XGenomics/cellranger#build-dependencies but I'm not sure what relies on that particular version.